cli/azd/pkg/project/framework_service_npm.go

```diff
@@ -106,3 +109,16 @@ func (np *npmProject) Build(
 		},
 	)
 }
+
+const cNodeModulesName = "node_modules"
+
+func excludeNodeModules(path string, file os.FileInfo) bool {
```

Comment:
```suggestion
func excludeNodeModules(path string, fileInfo os.FileInfo) bool {
```

Nit: since it's a file or a folder I would rename it to something more generic.

Reply:
Consider something more simple like `entry.IsDir() && entry.Name == "cNodeModulesName"`